### PR TITLE
OSDOCS-2711 PV note for Windows

### PR DESCRIPTION
**Pull request by codyhoag**
https://issues.redhat.com/browse/OSDOCS-2711

Preview: https://deploy-preview-38157--osdocs.netlify.app/openshift-enterprise/latest/nodes/containers/nodes-containers-projected-volumes#nodes-containers-projected-volumes-about_nodes-containers-projected-volumes

### CODE REVIEW

**Comment by codyhoag** [comment]
@rrasouli can you verify this? Thanks!

**Comment by codyhoag** [comment]
@sgaoshang @rrasouli resurfacing this. Can you give a QE ack? Thanks!

**Comment by rrasouli** [comment]
That looks fine to me
/lgtm

File: modules/nodes-containers-projected-volumes-about.adoc [hunk 1]
```diff
@@ -12,6 +12,13 @@ so that I can synthesize a single directory with various sources of information;
 * populate a single volume with the keys from multiple secrets, config maps, and with downward API information,
 explicitly specifying paths for each item, so that I can have full control over the contents of that volume.
 
+[IMPORTANT]
+====
+When the `RunAsUser` permission is set in the security context of a Linux-based pod, the projected files have the correct permissions set, including container user ownership. However, when the Windows equivalent `RunAsUsername` permission is set in a Windows pod, the kubelet is not able to correctly set ownership on the files in the projected volume.
```
Comment (stevsmit) [inline modules/nodes-containers-projected-volumes-about.adoc:17]: s/ not able/ unable? 

Entirely up to you.

**Pull request merged**